apps/main/src/loan/components/PageLlamaMarkets/cells/MarketTitleCell/TokenPair.tsx

```diff
@@ -0,0 +1,37 @@
+import { AssetDetails, LlamaMarket } from '@/loan/entities/llama-markets'
+import { getImageBaseUrl } from '@ui/utils'
+import Box from '@mui/material/Box'
+import Tooltip from '@mui/material/Tooltip'
+import TokenIcon from '@/loan/components/TokenIcon'
+import { SizesAndSpaces } from '@ui-kit/themes/design/1_sizes_spaces'
+import { ChainIcon } from '@ui-kit/shared/icons/ChainIcon'
+import { ReactNode } from 'react'
+import type { SxProps, Theme } from '@mui/material/styles'
+
+type TokenPairProps = Pick<LlamaMarket, 'assets' | 'chain'>
+
+const { IconSize } = SizesAndSpaces
+
+const TooltipBox = ({ title, children, sx }: { title: string; children: ReactNode; sx: SxProps<Theme> }) => (
+  <Box sx={{ position: 'absolute', ...sx }}>
+    <Tooltip title={title} placement="top">
+      <Box>{children}</Box>
+    </Tooltip>
+  </Box>
+)
+
+const TokenBox = ({ coin: { address, chain, symbol }, sx }: { coin: AssetDetails; sx: SxProps<Theme> }) => (
```

Comment:
Will generalize this and move to curve-ui-kit myself for the token selector modal PR

Reply:
Thanks!